routers/web/user/setting/oauth2_common.go

```diff
@@ -40,7 +41,7 @@ func (oa *OAuth2CommonHandlers) AddApp(ctx *context.Context) {
 	// TODO validate redirect URI
 	app, err := auth.CreateOAuth2Application(ctx, auth.CreateOAuth2ApplicationOptions{
 		Name:               form.Name,
-		RedirectURIs:       []string{form.RedirectURI},
+		RedirectURIs:       strings.Split(form.RedirectURIs, ","),
```

Comment:
We need to handle possible space, some people may input `url1, url2`.

Reply:
That should not happen with textarea/newlines I think